privacy-security-explainer.md

```diff
@@ -0,0 +1,68 @@
+# Privacy and security
+
+The WebXR Device API enables developers to build content for AR and VR hardware that uses one or more sensors to infer information about the real world, and may then present information about the real world either to developers or directly to the end user. In such systems there are a wide range of input sensor types used (cameras, accelerometers, etc), and a range of real-world data generated. This data is what allows web developers to author WebXR-based experiences. It also enables developers to infer information about people using their sites such as profiling, fingerprinting, and input sniffing. Due to the nature of the Web, WebXR has a higher responsibility to protect users from malicious data usage than XR experiences delivered through closed ecosystem app stores.
```

Comment:
> It also enables developers to infer information about people using their sites such as profiling, fingerprinting, and input sniffing.

Silly nit: I got caught on this sentence and had to read it several times over before it clicked. I initially read it as "developers, using their sites such as profiling [...], infer information about people."

What about simply:

> It also enables developers to infer information about **users** such as ...